colrev/ops/built_in/search_sources/eric.py

```diff
@@ -54,42 +76,183 @@ def validate_source(
             f"Validate SearchSource {source.filename}"
         )
 
-        if "query_file" not in source.search_parameters:
-            raise colrev_exceptions.InvalidQueryException(
-                f"Source missing query_file search_parameter ({source.filename})"
-            )
-
-        if not Path(source.search_parameters["query_file"]).is_file():
-            raise colrev_exceptions.InvalidQueryException(
-                f"File does not exist: query_file {source.search_parameters['query_file']} "
-                f"for ({source.filename})"
-            )
-
         search_operation.review_manager.logger.debug(
             f"SearchSource {source.filename} validated"
         )
-
     @classmethod
     def heuristic(cls, filename: Path, data: str) -> dict:
         """Source heuristic for ERIC"""
 
-        result = {"confidence": 0.0}
+        result = {"confidence": 0.1}
 
         # Note : no features in bib file for identification
 
         return result
+    
 
+    def search_split(search) -> str:
```

Comment:
This should be `__search_split(...)` because it is only used within EricSearchSource.